src/components/Nav/Modal.js

```diff
@@ -9,8 +9,18 @@ import {
 } from './CategoryPage/categoryList';
 import '../../styles/mixin.scss';
 import './Modal.scss';
+import Cart from '../../pages/Cart/Cart';
 
 const Modal = ({ closeModal, listCheck, listId }) => {
+    const [iscartModal, setIsCartModal] = useState(false);
```

Comment:
```suggestion
    const [isCartModal, setIsCartModal] = useState(false);
```
iscartModal=> isCartModal

Reply:
넵 수정했습니다.